crates/autopilot/src/solvable_orders.rs

```diff
@@ -590,12 +590,16 @@ async fn filter_unsupported_tokens(
     Ok(orders)
 }
 
-fn filter_limit_orders_with_insufficient_sell_amount(mut orders: Vec<Order>) -> Vec<Order> {
-    // Unwrap because solvable orders always have a surplus fee.
-    orders.retain(|order| match &order.metadata.class {
-        OrderClass::Limit(limit) => order.data.sell_amount > limit.surplus_fee.unwrap(),
-        _ => true,
-    });
+fn filter_fok_limit_orders_with_insufficient_sell_amount(mut orders: Vec<Order>) -> Vec<Order> {
```

Comment:
Do you have a suggestion for a better name? With the current name, all the doc comment can code is repeat the name. It could explain "fok" I guess. I intentionally started using "fok" instead of writing out fill_or_kill everywhere in a previous PR because I felt that makes it easier to read.